webpack/scenes/ContentViews/Details/DetailsContainer.js

```diff
@@ -31,11 +31,6 @@ const DetailsContainer = ({ children, cvId, isOpen }) => {
 DetailsContainer.propTypes = {
   children: PropTypes.element.isRequired,
   cvId: PropTypes.number.isRequired,
-  isOpen: PropTypes.bool,
-};
-
-DetailsContainer.defaultProps = {
-  isOpen: true,
```

Comment:
This variable wasn't being used, so was removed.

Reply:
ummm This is a mystery to me, I went back through the git history and this was NEVER really used. 
Perhaps the idea was that this details view eventually be accessible via a modal?  
But in its current implementation  they might as well have just done `const isOpen = true; `at the top of this functional scope.